code/modules/projectiles/guns/projectile/boltgun/roe_sika.dm

```diff
@@ -1,39 +1,3 @@
-/obj/item/gun/projectile/boltgun/light
```

Comment:
Remember to remove it as well from the random gun pile spawn on `code/game/objects/random/guns.dm`

Reply:
Handled it. Roe had its path changed to boltgun/light instead of boltgun/light_wood (since why specify it's wood if the polymer, aka the uglier one, is gone)